src/parser/mod.rs

```diff
@@ -3947,20 +3951,31 @@ impl<'a> Parser<'a> {
         Ok(values)
     }
 
+    //Parse a 'FROM' statment. support trailing comma
+    pub fn parse_from(&mut self) -> Result<Vec<TableWithJoins>, ParserError> {
```

Comment:
```suggestion
    /// Parse a list of [TableWithJoins]
    fn parse_table_with_joins(&mut self) -> Result<Vec<TableWithJoins>, ParserError> {
```
Thinking no need to make this function public. And `parse_from` sounded generic since the actual `FROM` keyword isn't part of the parsed elements